PULL_REQUEST_TEMPLATE

```diff
@@ -5,33 +5,33 @@ Please include a summary of the change and which issue is fixed. Please also inc
 List any dependencies that are required for this change.
 -->
 
+This pull request fixes issue #<number>
+
 ## Submitter Checklist:
 
 - [ ] Submitted a [ticket](https://github.com/brave/brave-ios/issues) for my issue if one did not already exist.
-- [ ] My patch or PR title has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!` (or `No Bug: <message>` if no relevant ticket)
 - [ ] *Unit Tests* are updated to cover new or changed functionality
 - [ ] User-facing strings use `NSLocalizableString()`
-- [ ] New files have MPL-2.0 license header.
+- [ ] Screenshots of any new designs changes are attached.
 
 
 ## Test Plan:
-
 <!-- Any useful notes for reviewer explaining how best to test and verify. -->
 
-### Screenshots:
 
+### Screenshots:
 <!-- If your patch includes user interface changes that you would like to suggest or that you would like UX to look at, please include them here. -->
 
 
 ## Reviewer Checklist:
 
-- [ ] PR is linked to an issue via [Zenhub](https://www.zenhub.com/extension).
-- [ ] Issues are assigned to at least one epic.
 - [ ] Issues include necessary QA labels:
-  - [ ] `QA/(Yes|No)`
-  - [ ] `release-notes/(include|exclude)`
-  - [ ] `bug` / `enhancement`
+  - `QA/(Yes|No)`
+  - `release-notes/(include|exclude)`
+  - `bug` / `enhancement`
 - [ ] Necessary security reviews have taken place.
```

Comment:
For `[ ] Necessary security reviews have taken place.`

OR in the ReadMe of the repo, we should specify that we open security reviews at https://github.com/brave/internal
or somewhere. I got lost on this one before :D

Reply:
Yeah, internal is private, so only we can submit sec tickets, but maybe providing a link would be helpful here, will add.